velox/dwio/orc/test/CMakeLists.txt

```diff
@@ -25,3 +25,6 @@ target_link_libraries(
   gtest
   gtest_main
   gmock)
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/examples
```

Comment:
You should not need that if you run the tests from the root directory of the project

Reply:
I guess so, and it is nice to do the same as `ParquetReaderTest` in #8460, as someone may run the tests in IDE, which seems running from the build directory.